### ISSUE TITLE
Font Loader does not work properly under vue-cli(webpack 4)

### ISSUE THREAD

**Issue by MosasoM** [mrdoob/three.js#19043]
##### Description of the problem

When I try to load a font like a [documentation](https://threejs.org/docs/#api/en/geometries/TextGeometry),it returns "Uncaught Syntax Error"

```
let loader = new THREE.FontLoader();
            loader.load("[font path]",
                function(font){
                    let geo = new THREE.TextGeometry("hoge",{
                        font:font,
                        size:80,
                        height:5,
                        curveSegments: 12,
                        bevelEnabled: true,
                        bevelThickness: 10,
                        bevelSize: 8,
                        bevelOffset: 0,
                        bevelSegments: 5
                    });
                }
            );
```

```
VM263:1 Uncaught SyntaxError: Unexpected string in JSON at position 1
    at JSON.parse (<anonymous>)
    at Object.eval [as onLoad] (three.module.js?5a89:41233)
    at XMLHttpRequest.eval (three.module.js?5a89:35829)
```

I tried some, and it finally works with below

```
import genfont from "@/assets/fonts/gentilis_bold.typeface.json"

let f = new THREE.Font(genfont)
let geo = new THREE.TextGeometry("hoge",{
                font:f,
                size:80,
                height:5,
                curveSegments: 12,
                bevelEnabled: true,
                bevelThickness: 10,
                bevelSize: 8,
                bevelOffset: 0,
                bevelSegments: 5
});
```

When I passed json obj to THREE.FontLoader() like code below, it returns same "Unexpected Syntax Error".
I found that under my environment, json file was somehow automatically parsed, and the error was caused by JSON.parse() handling not a json-style text but a json object. 

```
import genfont from "@/assets/fonts/gentilis_bold.typeface.json"

// genfont is json object 

let loader = new THREE.FontLoader();
loader.load(genfont,
                function(font){
                    let geo = new THREE.TextGeometry("hoge",{
                        font:font,
                        size:80,
                        height:5,
                        curveSegments: 12,
                        bevelEnabled: true,
                        bevelThickness: 10,
                        bevelSize: 8,
                        bevelOffset: 0,
                        bevelSegments: 5
                    });
```

I'm not sure what' a good solution to this. (Add an extra error handling to [FontLoader.js](https://github.com/mrdoob/three.js/blob/master/src/loaders/FontLoader.js) will work, but I don't think this is a good design...)

##### Three.js version

- [ ] Dev
- [x] r115
- [ ] ...

##### Browser

- [ ] All of them
- [x] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [ ] Windows
- [x] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by mrdoob** [comment]
Isn't this an issue with vue-cli instead?

**Comment by MosasoM** [comment]
I found that same problem was reported under typescript+webpack4+three.js. ([this](https://qiita.com/ankd/items/23e985b782dc1f7de330) is written in japanese)

So,  I think this is issue with webpack4 or three.js...

Where I wrote,

> I'm not sure what' a good solution to this. 

it includes that I'm not sure this problem should be addressed in three.js or others.

If you don't think this is a issue with three, I follow you.
